packages/back-end/src/models/SdkConnectionModel.ts

```diff
@@ -149,15 +147,14 @@ export async function createSDKConnection(params: CreateSDKConnectionParams) {
     key: generateSDKConnectionKey(),
     proxy: addEnvProxySettings({
       enabled: !!proxyEnabled,
-      host: proxyHost || "",
-      hostExternal: proxyHost || "",
+      host: proxyHost || PROXY_HOST_INTERNAL || PROXY_HOST_PUBLIC,
+      hostExternal: proxyHost || PROXY_HOST_PUBLIC || PROXY_HOST_INTERNAL,
```

Comment:
Is this needed?  We're calling `addEnvProxySettings` here, so this should be added already there